src/main/scala/temple/generate/database/ast/Condition.scala

```diff
@@ -0,0 +1,11 @@
+package temple.generate.database.ast
+
+/** AST Implementation of all boolean conditions supported in Templefile */
+sealed trait Condition
+
+object Condition {
```

Comment:
Do we want to add `IsNull(column)`? Note that `Inverse(IsNull(column))` is usually rendered as `column IS NOT NULL`.